src/api/base.ts

```diff
@@ -1,4 +1,5 @@
 import Axios from 'axios';
 
-const baseUrl = 'http://192.168.0.122:3100';
+// const baseUrl = 'http://192.168.0.122:3100';
```

Comment:
comment?

Reply:
On purpose. We might have to switch between both for dev purposes.